Go/Go.sublime-syntax

```diff
@@ -374,28 +374,49 @@ contexts:
       scope: variable.language.anonymous.go
       pop: 1
 
-  # https://golang.org/ref/spec#Operators_and_punctuation
+  # https://go.dev/ref/spec#Operators_and_punctuation
   #
   # Note: pipe "|" is not always bitwise. In type unions
   # it should be scoped differently.
   match-operators:
-    - match: '<<=|>>=|&\^=|&=|\^=|\|=|%=|\+=|-=|\*=|/=|\+\+|--'
+    - match: '<<=|>>=|&\^=|&=|\^=|\|=|%=|\+=|-=|\*=|/='
       scope: keyword.operator.assignment.augmented.go
     - match: '<-|:='
       scope: keyword.operator.assignment.go
-    - match: '&\^|<<|>>'
-      scope: keyword.operator.bitwise.go
-    - match: '==|!=|<=|>=|<|>'
-      scope: keyword.operator.comparison.go
+    - match: '(&\^)|(<<)|(>>)'
+      captures:
+        1: keyword.operator.bitwise.and_not.go
+        2: keyword.operator.bitwise.shift.left.go
+        3: keyword.operator.bitwise.shift.right.go
+    - match: '(==)|(!=)'
+      captures:
+        1: keyword.operator.comparison.equal.value.go
+        2: keyword.operator.comparison.not-equal.value.go
+    - match: '(<=)|(>=)|(<)|(>)'
+      captures:
+        1: keyword.operator.relational.less-equal.go
+        2: keyword.operator.relational.greater-equal.go
+        3: keyword.operator.relational.less.go
+        4: keyword.operator.relational.greater.go
```

Comment:
I don't think we need too detailed scopes.

Reply:
It would effect all syntaxes if we'd want to add that and the little value it provides is not worth the effort and possible performance implications.